### ISSUE TITLE
Failed prop type: Invalid prop `style.display` supplied to `Motion`.

### ISSUE THREAD

**Issue by liveresume** [souhe/reactScrollbar#66]
Failed prop type: Invalid prop `style.display` supplied to `Motion`.

When using:

```
            <ScrollArea
              style={{height: '100vh'}}
              verticalScrollbarStyle={{display: 'none'}}> ... 
```

**Comment by sizeight** [comment]
Same as issue #59

**Comment by brezhitskiy42** [comment]
How to fix this?
`Failed prop type: Invalid prop 'style.background' supplied to 'Motion'.`
My code:
`verticalScrollbarStyle={{ 'background': '#1d1f2d' }}`

**Comment by liveresume** [comment]
@sizeight Any fix in sight?

**Comment by souhe** [comment]
@sizeight did a PR. I left 2 small comments and after fixing that I'll merge it. It should fix this issue ;)

**Comment by DonovanCharpin** [comment]
Great @souhe 👍

**Comment by liveresume** [comment]
@souhe Thank You

**Comment by souhe** [comment]
Thanks to @sizeight it should be fixed since version v0.5.1 :)
